_posts/2024-12-12-Introducing-BMO-E2E.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Introducing Baremetal Operator end-to-end test suite"
+date: 2024-12-13
+draft: false
+categories: ["metal3", "cluster API", "provider", "edge"]
+author: Lennart Jern
+---
+
+In the beginning, there was metal3-dev-env. It could set up a virtualized
```

Comment:
link dev-env?

```suggestion
In the beginning, there was [metal3-dev-env](https://github.com/metal3-io/metal3-dev-env). It could set up a virtualized
```